### PR TITLE
Increase unit test coverage

### PR DESCRIPTION
**Pull request by saimonation**

### CODE REVIEW

File: cterasdk/core/users.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Users(BaseCommand):
     Portal User Management APIs
     """
 
-    _default_fields = ['name']
```
Comment (ygalblum) [inline cterasdk/core/users.py:15]: Why did you remove the leading underscore? It's there to tell that it is a private class member

Comment (saimonation) [reply cterasdk/core/users.py:15]: I am fine with either. In core/devices we use **default** - I wanted to align to that. But i'm fine with the other way around.

Comment (ygalblum) [reply cterasdk/core/users.py:15]: OK, leave it as you did
